src/core/server/http/http_tools.ts

```diff
@@ -36,6 +36,10 @@ export function getServerOptions(config: HttpConfig, { configureTLS = true } = {
     host: config.host,
     port: config.port,
     routes: {
+      cache: {
+        privacy: 'private',
+        otherwise: 'private, no-cache, no-store, must-revalidate',
+      },
```

Comment:
I see you added server_integration tests for it, but maybe we should also add a test for this in `src/core/server/http/integration_tests/router.test.ts` to check that 1/ this is the default value and 2/ route handlers can still override it?

Reply:
Good idea, will do!